docs/how-to-enable-new-languages.md

```diff
@@ -281,3 +281,138 @@ Once these are in place, you should be able to run `npm run develop` to view you
+        '_'
+      )
+  };
+});
+```
+
+Then open a PR to the CDN repo to add both the YAML and Day.js files for review.
+
+## Prep the News Repo for the New Language
+
+The [News repo](https://github.com/freeCodeCamp/news) pulls data from a Ghost instance, the files you added to the CDN, builds News, and deploys it.
+
+> [!WARN]
+> Pull requests to the news repo _must_ come from the same repo. You should not work off of a fork for this step.
+
+### Modify the Main Config File
+
+Clone the News repo and create a new branch.
+
+Open the `config/index.js` file to add the new language and configure the necessary values. There are a few objects and arrays to modify:
+
+- `locales`: This array contains the active and upcoming News languages. These are the values that are used in the `.env` file to choose the Ghost instance and UI to use for each build. Add the text name of the new language in lowercase to this array.
+- `localeCodes`: This object is a map of ISO codes for each language, and is used to configure i18next before building the UI. To add a new language, use the lowercase language name as the _key_ and the ISO 639-1 language code as the _value_.
+- `algoliaIndices`: This object is a map of Algolia indices for each language. To add a new language, use the lowercase language name as the _key_, and `news-` followed by the lowercase ISO 639-1 language code as the _value_.
+
+> [!NOTE]
+> If you are unsure about the string to use while setting `algoliaIndices`, send a message to Kris (@scissorsneedfoodtoo), or someone else with access to Algolia, and ask them to check.
+
+For example, if you are launching Dothraki News, here are what the objects / arrays above should look like:
+
+```js
+const locales = ['arabic', 'bengali', 'chinese', 'english', 'dothraki'];
+
+const localeCodes = {
+  arabic: 'ar',
+  bengali: 'bn',
+  chinese: 'zh',
+  english: 'en',
+  dothraki: 'mis'
+};
+
+const algoliaIndices = {
+  arabic: 'news-ar',
+  bengali: 'news-bn',
+  chinese: 'news-zh',
+  english: 'news',
+  dothraki: 'news-mis'
+};
+```
+
+### Add the i18next JSON Files for the New Language
+
+Next, go to the `config/i18n/locales` directory, create a new folder, and give it the name of the new language you're adding. For example, if you're launching Dothraki News, create a new folder named `dothraki`.
+
+Then go to the `english` directory, copy the JSON files, and paste them into your new folder.
+
+In your new folder, open the `serve.json` file and replace its contents with the following:
+
+```json
+{
+  "redirects": []
+}
+```
+
+Then commit and push your branch directly to the News repo.
+
+> [!NOTE]
+> You need to be on one of the teams with access to the News repo to push branches directly to News. Currently, only the dev, i18n, and staff teams are allowed to do this.
+
+Finally, open a PR for review.
+
+Once both your PRs to the CDN and News repo are merged, staff will create a couple more PRs to enable the new language and launch the site.
```

Comment:
Is the intention to merge these PRs whenever they've been through review or wait until you're ready to deploy?
```suggestion
Once both your PRs to the CDN and News repo have been approved, they can be merged. Deployment will be handled subsequently by the staff.
```
Given that deployment is out of scope, I don't think we need to give details.

Reply:
Sure, this suggestion looks good. The main thing I want to get across is that there are a few more steps before News can be launched for the new language, and it's still clear with this wording.